src/ipfsd-daemon.js

```diff
@@ -323,7 +327,8 @@ class Node {
       ),
       (config, cb) => {
         if (!key) {
-          return tryJsonParse(config, cb)
+          // TODO: this function didn't exist on the original codebase
+          // return tryJsonParse(config, cb)
```

Comment:
- [ ] Figure out why this was here in the first place.

Reply:
because the daemon process will return a json string back, not an object.

Reply:
@dryajov even if that is correct, the `tryJsonParse` function doesn't seem to exist anywhere in the codebase